contracts/contracts/vault/VaultCore.sol

```diff
@@ -217,27 +217,7 @@ contract VaultCore is VaultStorage {
         // date picture of total assets before allocating to strategies.
         for (uint256 i = 0; i < allStrategies.length; i++) {
             IStrategy strategy = IStrategy(allStrategies[i]);
-            address rewardTokenAddress = strategy.rewardTokenAddress();
-            if (rewardTokenAddress != address(0)) {
-                uint256 liquidationThreshold = strategy
-                    .rewardLiquidationThreshold();
-                if (liquidationThreshold == 0) {
-                    // No threshold set, always harvest from strategy
-                    IVault(address(this)).harvestAndSwap(allStrategies[i]);
-                } else {
-                    // Check balance against liquidation threshold
-                    // Note some strategies don't hold the reward token balance
-                    // on their contract so the liquidation threshold should be
-                    // set to 0
-                    IERC20 rewardToken = IERC20(rewardTokenAddress);
-                    uint256 rewardTokenAmount = rewardToken.balanceOf(
-                        allStrategies[i]
-                    );
-                    if (rewardTokenAmount >= liquidationThreshold) {
-                        IVault(address(this)).harvestAndSwap(allStrategies[i]);
-                    }
-                }
-            }
+            IVault(address(this)).harvestAndSwap(allStrategies[i]);
```

Comment:
Well this is much nicer! Way to make the code better!

Reply:
tnx

![e2870a557842b5a32324e83b2cdab540](https://user-images.githubusercontent.com/579910/147930342-821f5343-3137-4881-94f2-eaddf8cedf55.jpeg)

😄